oml/lightning/callbacks/metric.py

```diff
@@ -1,20 +1,31 @@
-from typing import Any, Optional
+from typing import Any, Iterable, Optional
 
+import numpy as np
 import pytorch_lightning as pl
+from neptune.new.types import File
 from pytorch_lightning import Callback
+from pytorch_lightning.loggers import NeptuneLogger, TensorBoardLogger
 from pytorch_lightning.utilities.types import STEP_OUTPUT
 
+from oml.analysis.visualisation import RetrievalVisualizer
+from oml.const import OVERALL_CATEGORIES_KEY
 from oml.interfaces.metrics import IBasicMetric
+from oml.metrics.embeddings import EmbeddingMetrics
 from oml.utils.misc import flatten_dict
 
 
 class MetricValCallback(Callback):
     def __init__(
         self,
         metric: IBasicMetric,
+        save_image_logs: bool = False,
         log_only_main_category: bool = True,
         loader_idx: int = 0,
         samples_in_getitem: int = 1,
+        image_top_k_per_metric: int = 3,
+        image_top_k_in_row: int = 5,
+        image_folder: str = "image_logs",
+        image_metrics_to_ignore: Iterable[str] = ("cmc",),
```

Comment:
I think that for simplicity of `MetricValCallback` we can move these new params to `const.py` and do not pass them to constructor

What I suggest:
- always sort predictions by map-5
- always show 5 rows (5 queries) and 5 predictions for each query
- image_folder -- always set to `image_logs`

So, I suggest to pass only `save_image_logs` as a parameter